cmd/kpod/run.go

```diff
@@ -136,5 +136,9 @@ func runCmd(c *cli.Context) error {
 		fmt.Printf("%s\n", ctr.ID())
 	}
 	wg.Wait()
+	if createConfig.rm {
+		_ = ctr.Stop()
```

Comment:
Don't need this, we're doing a force remove so it will be stopped in Remove if necessary

Reply:
yes just trying to be nice, but I guess we should rely on the --force